sandbox/main.lua

```diff
@@ -27504,9 +27508,9 @@ function tsarina(enemyFaction)
 		setWeaponStorage("EMP", 0):
 		setWeaponStorage("Homing", 0):
 		setWeaponStorage("HVLI", 100):
-		setHull(600):
-		setShields(100, 100, 100):
```

Comment:
Having a player ship with three shield arcs does not give it three shield engineering systems. This has been tried by others and it does not work.

Reply:
I think this was a display bug of github?...